pkg/skel/skel.go

```diff
@@ -190,6 +203,29 @@ func validateConfig(jsonBytes []byte) *types.Error {
 	return nil
 }
 
+func getCurrentThreadNetNSPath() string {
+	// /proc/self/ns/net returns the namespace of the main thread, not
+	// of whatever thread this goroutine is running on.  Make sure we
+	// use the thread's net namespace since the thread is switching around
+	return fmt.Sprintf("/proc/%d/task/%d/ns/net", os.Getpid(), unix.Gettid())
+}
+
+func checkNetNS(nsPath string) (bool, *types.Error) {
+	ns, err := netns.GetFromPath(nsPath)
+	if err != nil {
```

Comment:
We need to support CNI DEL for empty nsPath as well as already-deleted nsPath. So, paradoxically, I think we can return `true` here if `err != nil`.

Reply:
Do you mean we just pass the check and let plugins deal with empty plugins as well as already-deleted nsPath？